Pipfile

```diff
@@ -11,6 +11,7 @@ st-annotated-text = "*"
 streamlit-embedcode = "*"
 
 [dev-packages]
+pytest = "*"
```

Comment:
Shouldn't there be more here? Like pre-commit, black, etc?

Reply:
Yeah, there should.